core/templates/dev/head/css/oppia.css

```diff
@@ -1128,6 +1180,13 @@ md-card.oppia-gallery-tile {
   }
 }
 
+.oppia-alert-blocks {
```

Comment:
OK -- if you didn't mean to add it, I would suggest deleting it. The file diff at https://github.com/oppia/oppia/pull/1638/ should only reflect the changes you've added.